log/log.go

```diff
@@ -31,24 +35,47 @@ func NewLeveledLogger(l kit_log.Logger) Logger {
 // Valuer with their generated value for each call to its Log method.
 func With(logger Logger, keyvals ...interface{}) Logger {
 	return &ctLogger{
-		logger: kit_log.With(logger.ToGoKitLogger(), keyvals),
+		logger: kit_log.With(logger.ToGoKitLogger(), keyvals...),
+	}
+}
+
+// AllowLevel sets the log filtering according to the provided level
+func AllowLevel(logger Logger, level kit_level.Option) Logger {
+	return &ctLogger{
+		logger: kit_level.NewFilter(logger.ToGoKitLogger(), level),
+	}
+}
+
+// ConvertToLevel transform string value in level
+func ConvertToLevel(strLevel string) (kit_level.Option, error) {
+	switch strLevel {
+	case "debug":
+		return kit_level.AllowDebug(), nil
+	case "info":
+		return kit_level.AllowInfo(), nil
+	case "warn":
+		return kit_level.AllowWarn(), nil
+	case "error":
+		return kit_level.AllowError(), nil
```

Comment:
Might be simplier with a map ?
map[string]kit_level.Option {
    "debug": kit_level.AllowDebug(),
    "info": kit_level.AllowInfo(),
}